parm/config/config.eupd

```diff
@@ -11,6 +11,25 @@ echo "BEGIN: config.eupd"
 export ENKFUPDSH="$HOMEgsi/scripts/exglobal_enkf_update_fv3gfs.sh.ecf"
 export npe_enkf=$npe_eupd
 
-export NAM_ENKF="analpertwtnh=0.9,analpertwtsh=0.9,analpertwttr=0.9"
+# Use NAM_ENKF below for serial EnKF
+##export NAM_ENKF="analpertwtnh=0.9,analpertwtsh=0.9,analpertwttr=0.9"
+
+# LETKF specific settings with model space localization
+export modelspace_vloc=".true."  # model space localization
+export letkf_flag=".true."       # use LETKF instead of serial filter
+export getkf=".true."            # Gain form of LETKF (needed for model-space localization)
+export denkf=".true."            # EnKF approximation (beneficial since less spread removed by analysis)
+export nobsl_max=10000           # max number of obs in each LETKF volume (uses closest nobsl_max). can
```

Comment:
are we decreasing this because of runtime concerns?

Reply:
Cory:  these parameters are copied over from v16rt2.

Reply:
@yangfanglin yes sorry, I meant more for asking @RussTreadon-NOAA if we decided if we need to decrease from 10000 due to runtime concerns from testing Cathy has done.

Reply:
Good question.  Let's ask Cathy what the DA team decided.  I don't recall.